agenta-web/src/components/Playground/VariantRemovalWarningModal.tsx

```diff
@@ -38,12 +38,6 @@ const VariantRemovalWarningModal: React.FC<Props> = ({
         handleCloseModal()
     }
 
-    useEffect(() => {
```

Comment:
As a result of this change, deleting the last variant in an app does not result in navigation to the main menu